### DISCUSSION TITLE
Access language code from Faker instance

### DISCUSSION THREAD

**Discussion by matthewmayer** [faker-js/faker#1908] [General]
Am i missing something, is there a way to access the current language code from a prebuilt faker instance?

I was trying to print out a list of all locales and the value of a certain method like street address e.g.

```
af_ZA 012 Bernadette Bridge
ar 702 ساحة الصغير
az küçəsi Yaşar Əliyev, 090
cz Nad Pískovnou 017
```

i tried
```js
const {allFakers} = require("@faker-js/faker")
Object.values(allFakers).map(f=>f.location.streetAddress())
```
gets me the street addresses, but not the codes. The locale property of each faker throws an error.

i ended up with

```js
Object.entries(allFakers).map(entry=>{let [key,faker]=entry;return(key+" "+faker.location.streetAddress())})
```

which is a bit ugly. is there a nicer way to access the property on the Faker instance itself?

Similarly i'd expect something like

```js
let localFaker = fakerEN.helpers.arrayElement([fakerDE,fakerEN])
let name = localFaker.person.name()
console.log("name is "+name+" and their language is "+localFaker.localeCode)
```

**Comment by ST-DDT** [comment]
In v7 via faker.locale
In the next version you can use `allFakers.<locale>.person.firstName()`
We plan to move definitions.title to definitions.metadata.title and add more data there.
I cannot find the Issue or comment right now.
